### DISCUSSION TITLE
New demo release 11.14.0

### DISCUSSION THREAD

**Discussion by palexdev** [palexdev/MaterialFX#296] [General]
[Changelog](https://github.com/palexdev/MaterialFX/blob/main/CHANGELOG.md)

<hr /><em>This discussion was created from the release <a href='https://github.com/palexdev/MaterialFX/releases/tag/v11.14.0'>New demo release 11.14.0</a>.</em>

**Comment by Roccoxx** [comment]
Amazing!

When this will be upload in Maven Repository? https://mvnrepository.com/artifact/io.github.palexdev/materialfx

**Reply by palexdev** [reply]
> Amazing!
> 
> When this will be upload in Maven Repository? https://mvnrepository.com/artifact/io.github.palexdev/materialfx

It already is: [Maven](https://central.sonatype.com/artifact/io.github.palexdev/materialfx/11.14.0)

Little tip: mvnrepository is very slow in detecting new releases, use Sonatype/Maven Central instead ;)

**Comment by Ilirski** [comment]
Is it just me or does applying `Theme.DEFAULT` to the scene take a few seconds longer than applying `Theme.LEGACY`?

**Reply by palexdev** [reply]
It's normal, the DEFAULT theme is the one containing all the MaterialFX controls
The LEGACY one contains just a bunch of "original" JavaFX controls
